src/pages/courses/create.vue

```diff
@@ -1,66 +1,111 @@
 <template>
 	<div v-if="course">
-		<section class="section">
-			<h1>Kurs erstellen</h1>
-			<BaseInput
-				v-model="course.name"
-				label="Name"
-				type="text"
-				placeholder="Dream Team"
-				maxlength="30"
-			></BaseInput>
-			<BaseInput
-				v-model="course.description"
-				label="Beschreibung"
-				type="textarea"
-				placeholder="Everything you have to know"
-				maxlength="255"
-			></BaseInput>
-			<button class="button is-primary" @click="create()">Speichern</button>
-			<h1>{{ course.name }}</h1>
-			<p>{{ course.description }}</p>
-		</section>
+		<TemplateCourseWizard
+			:steps="stepList"
+			:current-step="0"
+			:course="course"
+			:user="user"
+			:teachers="teachers"
+			:classes="classes"
+			:students="students"
+			@course-creation-submit="create()"
+		/>
 	</div>
 </template>
 
 <script>
 import { mapState } from "vuex";
+import { mapGetters } from "vuex";
```

Comment:
you can combine those imports
```suggestion
import { mapState, mapGetters } from "vuex";
```